stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Channel.kt

```diff
@@ -53,6 +53,9 @@ import java.util.Date
  * in the events.
  * @param membership Represents relationship of the current user to this channel.
  * @param extraData A map of custom fields for the channel.
+ * @param cachedMessages The list of cached messages if the regular list does not contain the newest loaded messages.
```

Comment:
Makes sense for cached messages. As for inside search it has some ties with it but we can see to rename it.

Reply:
I would go with `cachedLatestMessages` so that implies it is a cache and might be cleared at some point. For `latestMessages` i would expect them always to return something.